tket/src/Architecture/CMakeLists.txt

```diff
@@ -24,12 +24,16 @@ add_library(tket-${COMP}
     ArchitectureMapping.cpp
     BestTsaWithArch.cpp
     DistancesFromArchitecture.cpp
-    NeighboursFromArchitecture.cpp)
+    NeighboursFromArchitecture.cpp
+    SubgraphMonomorphisms.cpp
```

Comment:
I will probably just delete them (but keep and adapt the test), since I now realise that the Architecture class by itself is not enough - you use different vertex types (Node and Qubit) for the pattern and target graphs.

That's why src\Placement\MonomorphismCalculation.cpp and src\Placement\RelabelledGraphWSM.hpp are now replacing them.